src/main.rs

```diff
@@ -3,50 +3,43 @@
 #[macro_use]
 extern crate error_chain;
 extern crate blacklung;
-extern crate docopt;
-extern crate rustc_serialize;
 #[macro_use]
 extern crate slog;
+extern crate toml;
 
 use blacklung::server;
-use docopt::Docopt;
-
-const USAGE: &'static str = "
-Blacklung server.
-
-Usage:
-    blacklung [--port=<port>]
-    blacklung (-h | --help)
-
-Options:
-    -h --help       Show this screen.
-    --port=<PORT>   Port to bind to [default: 12345].
-";
-
-#[derive(Debug, RustcDecodable)]
-struct Args {
-    flag_port: u16,
-}
 use blacklung::logging;
+use blacklung::cfg;
+use std::process;
+
+fn handle_error<T, V, E>(f: T) -> V
```

Comment:
Not anything we can use now but just for reference... https://internals.rust-lang.org/t/rfc-mentoring-opportunity-permit-in-main/4600